File: backend/app/core/legal.py
"""Single source of truth for the AGB version currently in force.

Keep in sync with frontend/src/legal.ts's AGB_VERSION and AgbPage.tsx's
"Stand:"-date. Bumping this makes every account whose stored
agb_accepted_version differs see the AGB-Gate again on next login
(frontend/src/routes/AgbGate.tsx).
"""

CURRENT_AGB_VERSION = "2026-09-23"

File: frontend/src/legal.ts
// Single source of truth for the AGB version currently in force — bump this
// and AGB_VERSION_LABEL (and backend/app/core/legal.py's CURRENT_AGB_VERSION)
// whenever the AGB text changes materially; AgbPage.tsx shows the label.
// Every account whose stored agb_accepted_version differs sees the AGB-Gate
// again on next login (see routes/AgbGate.tsx).
export const AGB_VERSION = '2026-09-25'
export const AGB_VERSION_LABEL = '25. September 2026'
